test/dialogflow-app-test.js

```diff
@@ -3670,4 +3670,42 @@ describe('DialogflowApp', function () {
       expect(app.getPackageEntitlements()).to.be.null;
     });
   });
+
+  /**
+   * Describes the behavior for DialogflowApp askForUpdatePermission method in v1.
+   */
+  describe('#followUpEvents', function () {
+    let mockRequest, app;
+
+    beforeEach(function () {
+      mockRequest = new MockRequest(headerV1, dialogflowAppRequestBodyLiveSession);
```

Comment:
Change `headerV1` to `headerV2`, this might allow you to use camelCase in your tests, rather than snake_case

Reply:
ahhhh that's it! I was going mad.. thanks for your quick reply. Sorry for hacking the test, I didn't feel well when I did the commit.

Thank you for making such an amazing product.